internal/generator/vector/output/kafka/kafka.go

```diff
@@ -150,7 +150,7 @@ func TLSConf(o logging.OutputSpec, secret *corev1.Secret) []Element {
 
 		if security.HasPassphrase(secret) {
 			pp := security.Passphrase{
-				PassphrasePath: security.SecretPath(o.Secret.Name, constants.Passphrase),
+				KeyPass: security.GetFromSecret(secret, constants.Passphrase),
```

Comment:
@vimalk78 trying to remember here, did we switch to a configmap and need to ensure the passphrase does not end up in plane text?